app/javascript/gobierto_data/webapp/components/landingviz/VisualizationsGrid.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div
+    v-if="publicVisualizations.length"
+    class="gobierto-data-visualization--grid"
+  >
+    <template v-for="{ items, config, name, id, columns, slug, datasetName } in publicVisualizations">
+      <router-link
+        :key="id"
+        :to="`/datos/${slug}/v/${id}`"
+        class="gobierto-data-visualizations-name"
+      >
+        <VisualizationsSlot :name="name">
+          <template v-slot:visualization>
+            <Visualizations
+              v-if="items"
+              :items="items"
+              :object-columns="columns"
+              :config="config"
+            />
```

Comment:
Doing what I suggested before, this would become:

```suggestion
        <VisualizationsSlot>
          <template v-slot:title>{{ name }}</template>
          
          <Visualizations
              v-if="items"
              :items="items"
              :object-columns="columns"
              :config="config"
            />
```

You may take a look at the Dropdown vue-component for this kind of things, since the logic is quite similar